backend/Controllers/LocationController.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Services;
+
+namespace WhaleSpotting.Controllers {
+  [ApiController]
+  [Route("/locations")]
+  public class LocationController : ControllerBase {
+    private readonly ILocationService _locations;
+
+    public LocationController(
+      ILocationService locations
+    ) {
+      _locations = locations;
+    }
+  }
```

Comment:
I know somewhere else in the codebase I accidentally had 2 spaces per tab instead of 4, but could you change it back to 4 for this and the other 2 files? Thanks and sorry!
```suggestion
    [ApiController]
    [Route("/locations")]
    public class LocationController : ControllerBase {
          private readonly ILocationService _locations;

          public LocationController(
                ILocationService locations
          ) {
                _locations = locations;
          }
    }
```